packages/boxel/addon/components/boxel/modal/usage.hbs

```diff
@@ -10,6 +10,7 @@
   @isOpen={{this.isOpen}}
   aria-labelledby="boxel-modal-usage-example-id"
   style={{css-var
+    boxel-modal-max-width=this.maxWidth
     boxel-modal-offset-top=this.offsetTop
     boxel-modal-offset-left=this.offsetLeft
     boxel-modal-offset-right=this.offsetRight
```

Comment:
If we do this:

```
style={{html-safe (
      concat (css-var
        boxel-modal-offset-top=this.offsetTop
        boxel-modal-offset-left=this.offsetLeft
        boxel-modal-offset-right=this.offsetRight
      )
      "; "
      (if this.maxWidthOverride (css-var boxel-modal-max-width=this.maxWidthOverride) "")
    )
  }}
  ```
  
  We can avoid having two inputs affect the `this.maxWidthOverride` value.

Reply:
👍🏻 thanks, I incorporated that in the merge commit 13728f0